### PR TITLE
Neuburg 14 refactor and clean up

### PR DESCRIPTION
**Pull request by michael-markl**

### CODE REVIEW

File: src/modules/app/containers/RouterFragment.js [hunk 1]
```diff
@@ -2,26 +2,17 @@ import { Fragment } from 'redux-little-router'
 import React from 'react'
 import PropTypes from 'prop-types'
 import RouteConfig from '../RouteConfig'
-import { connect } from 'react-redux'
 import LivingFormPage from '../../../routes/living-form/containers/LivingFormPage'
-import LivingPage from '../../../routes/living/containers/LivingPage'
+import LivingPage from '../../../routes/categories/containers/CategoriesPage'
```
Comment (steffenkleinle) [inline src/modules/app/containers/RouterFragment.js]: To stay consistent we should import it as CategoriesPage then.

File: src/modules/app/containers/RouterFragment.js [hunk 2]
```diff
@@ -2,26 +2,17 @@ import { Fragment } from 'redux-little-router'
...
 import LivingManageOfferPage from '../../../routes/living-manage-offer/containers/LivingManageOfferPage'
```
Comment (steffenkleinle) [inline src/modules/app/containers/RouterFragment.js]: Wouldn't `LivingOfferPage` and `living-offer` as a name be enough? And why do we ust kebap-case here?

**Comment by steffenkleinle** [comment]
I think NEUBURG-9 would be done with this PR.

File: package.json [hunk 3]
```diff
@@ -12,107 +12,104 @@
     "classnames": "~2.2.5",
```
Comment (maxammann) [inline package.json:12]: I think we should do the package upgrades in sync with the integreat webapp. So we stuble across issues only once.

**Review by maxammann — APPROVED** [review]

File: src/modules/app/containers/RouterFragment.js (hunk 2 above)
Comment (michael-markl) [reply src/modules/app/containers/RouterFragment.js]: Where do we use kebab-case? Folders are all in kebab-case

File: src/modules/app/containers/RouterFragment.js (hunk 1 above)
Comment (michael-markl) [reply src/modules/app/containers/RouterFragment.js]: Right, thanks

File: package.json (hunk 3 above)
Comment (michael-markl) [reply package.json:12]: Yes, we can do it in sync from now on

**Pull request merged**